torch_xla/csrc/tensor.h

```diff
@@ -103,6 +103,10 @@ class XLATensor {
   static ir::Value GetDeviceDataIrValue(const at::Scalar& value,
                                         xla::PrimitiveType type,
                                         const Device& device);
+  // Use with caution, constant will cause more frequent recompilation
+  // compared to the device_data.
+  static ir::Value GetIrValueForConstant(const at::Scalar& value,
```

Comment:
Nit. Should we comment the function def with this info, too, at line 725?

Reply:
I think we usually don't repeat the comment in the `cpp` file